x-pack/elastic-agent/pkg/artifact/install/zip/zip_installer.go

```diff
@@ -40,6 +40,12 @@ func (i *Installer) Install(programName, version, installDir string) error {
 		return err
 	}
 
+	// cleanup install directory before unzip
+	_, err = os.Stat(installDir)
+	if err == nil || os.IsExist(err) {
+		os.RemoveAll(installDir)
```

Comment:
We ignore the error here. What should we do in case of an error? Same further above.

Reply:
Ignoring it, means that the extract will still try, and it will fail. The error will be more clear that extraction failed, vs an error to remove a directory when trying to perform an extract.